<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmConsultaAcervo</class>
 <widget class="QWidget" name="frmConsultaAcervo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>637</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="pesquisarLabel">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Pesquisar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="pesquisarLineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="pesquisarPushButton">
       <property name="text">
        <string>Pesquisar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="filtroDePesquisaLabel">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Filtro de Pesquisa</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="filtroDePesquisaComboBox">
       <item>
        <property name="text">
         <string>Titulo</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Autor</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Editora</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ISBN</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Codigo</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Nome</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
